src/main/java/uk/gov/justice/laa/crime/cfecrime/controllers/CfeCrimeController.java

```diff
@@ -27,14 +28,27 @@ public class CfeCrimeController {
             )
     )
     @PostMapping(produces = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity<CfeCrimeResponse> invoke(@Parameter(
+    public ResponseEntity<Object> invoke(@Parameter(
```

Comment:
This looks like a very unfortunate side-effect of the error handling. I think there is a better way that allows just the 2 valid responses